content/tutorial-air-quality-analysis.md

```diff
@@ -0,0 +1,251 @@
+---
+jupytext:
+  formats: ipynb,md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.11.5
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Analysis of air pollution levels before and after lockdown in Delhi, India
+
+
+## What you'll do
+
+Calculate Air Quality Indices (AQI) and perform paired t-test on them.
+
+## What you'll learn
+
+- You'll compute moving averages
+
+- You'll calculate Air Quality Index (AQI)
+
+- You'll perform a paired t-test and find the t and p values
+
+- You'll learn to interpret these values
+
+
+## What you'll need
+
+- SciPy and Matplotlib installed in your environment
+
+- Basic understanding of statistical terms like population, sample, mean, standard deviation etc.
+
+
+***
+
+```{code-cell} ipython3
+# import necessary libraries
+
+import numpy as np
+from scipy.stats import t
+import matplotlib.pyplot as plt
+
+%matplotlib inline
+```
+
+## Building the dataset
+
+We will import hourly measurements of standard pollutants: Particulate Matter (PM 10 and PM 2.5),
+Nitrogen dioxide (NO2), ammonia (NH3), sulfur dioxide (SO2), carbon monoxide (CO), and ozone (O3), from a *.csv file
+to ndarrays. This data has been collected in the period of 31/05/2019 to 30/06/2020.
```

Comment:
Can you include a source link for the data? Is it publicly available?

Reply:
Also, it might be nicer to have some small paragraph on why this data is relevant for measuring air quality - there's usually some blurb when these kinds of datasets are made public so it may be just a matter of citing it here.

Reply:
Sure, I can do that. I've used a condensed version of a publicly available dataset; I'll mention that.